app/views/customers/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= @customer.name %>'s Shopping List</h1>
+<div class="grocery-list">
+  <% @customer.items.each do |item| %>
+    <p>Item: <%= item.name %></p>
+    <p>Cost: <%= item.price %></p>
+    <p>Store: <%= item.supermarket.name %></p></br>
+  <% end %>
+</div>
+<div class="add-item-form">
+  <h4>Missing something? Look it up!</h4>
+  <%= form_with url: "/customers/#{@customer.id}", method: :post do |form| %>
```

Comment:
Don't forget to disable turbo with `data: {turbo: false}` just so you don't have weird behavior when running this code locally. For a more RESTful path, I'd send it to `"/customers/:customer_id/items"` to indicate that it's going to create the relationship between those 2 resources. 

Also, this is a minor semantics thing, but this search form is actually used to add an item to a customer, not to search for an item. The user needs to already know the item ID they want to add to a user's profile to fill out this form. Just mentioning to make sure that US was clear because your label suggests this is a searching form.